src/views/wallet/UserAssetsScreen.tsx

```diff
@@ -1,7 +1,51 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { Row, Col, Table } from 'antd'
+import DynamicCoin from '../../components/shared/CoinIcons/DynamicCoin'
 
-export default function UserAssetsScreen () : JSX.Element {
+type UserAssetType = {
+  _id: string;
+  free: number;
+  frozen: number;
+  locked: number;
+  symbol: string;
+  name: string;
+  value: string;
+}
+
+// Dummy data
+const UserAssets:UserAssetType[] = [
+  { _id: '1', free: 99, frozen: 11, locked: 21, symbol: 'BNB-JST', name: "Binance", value:"0.99"},
+  { _id: '2', free: 1034, frozen: 38, locked: 101, symbol: 'RUNE-1E0', name: "Rune", value:"0.25"}
+]
+
+const UserAssetsScreen: React.FC = (): JSX.Element => {
+  const [assets, setAssets] = useState<UserAssetType[]>([])
+
+  async function setData() {
```

Comment:
[nitpick] I think you can remove `async` here, since we will handle all business logic to get data in a service or so.